symplyphysics/core/geometry/line.py

```diff
@@ -0,0 +1,20 @@
+from sympy import symbols, Point, Line, Expr, solve
+
+def two_point_function(p1: Point, p2: Point, x: Expr) -> Expr:
+    """Constructs a linear function of ``x`` using two points ``p1`` and ``p2``.
+    
+    Returns ``NotImplemented`` if the line equation does not depend on ``y``.
+    """
+
+    xsym = x
```

Comment:
Don't save to temporary variable.

Reply:
It seemed reasonable to me to have `x` as a parameter of this function. Maybe `x_sym` can be fine too.

Reply:
You can keep it `x`. But rename temporary `x, y` symbols.